handler.js

```diff
@@ -340,3 +357,9 @@ function getObjectDefault(obj, key, defaultValue) {
     var value = obj[key];
     return value != undefined ? value : defaultValue;
 }
+
+function getKeyValuePairs(str, del = ';'){
+    if(str){
```

Comment:
Why test for empty/null? Seems the function just shouldn't be called...

Reply:
removing the function since we don't need to reuse it. moving the test empty/null up.